frontend/src/ui/Blocks/Series.tsx

```diff
@@ -160,7 +184,7 @@ const ReadySeriesBlock: React.FC<ReadyProps> = ({
 type Event = SeriesBlockSeriesData$data["events"][0];
 
 const compareNewToOld = compareByKey((event: Event): number => (
-    new Date(event.created).getTime()
+    new Date(event.syncedData?.startTime ?? event.created).getTime()
```

Comment:
I think this change is a good idea, but we should use this definition of "when event?" everywhere (unless we specifically talk about "creation date"). In particular, and most notably, the relative time shown in `GridTile` should use this definition. Otherwise you can end up with a list of videos where the relative times are definitely not sorted, confusing the user. 

I guess the only other place where it's relevant is the video page. But this will be redesigned a bit anyway, so it's probably fine if you just fix the "relative time" in `GridTile` in this PR.